vrk-kpa/suomidigi
# Suopa 475 approval queue

TO TEST:
- make fresh
- Go to http://suomidigi.fi.docker.amazee.io/en/admin/content/approval-queue and check for the queue.
